app/assets/css/vendor-override.css

```diff
@@ -445,3 +445,90 @@ input:-webkit-autofill {
 .tippy-content {
   padding: 0;
 }
+
+/* react-datetime-picker */
+/* https://github.com/wojtekmaj/react-datetime-picker#custom-styling */
+
+/*
+  library css is overriden by `.widget .widget-body button`
+  so we have to force margin: 0
+*/
+.react-calendar button {
```

Comment:
what about using css modules? so in ScheduledTimeField use `className={styles.datePicker}` and `import styles from './ScheduledTimeField.module.css`?